examples/pxScene2d/src/jsbindings/rcvrcore/rpcContext.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Created by tjcarroll2
+ * on 2/6/16.
+ */
+
+function RPCContext(theScene) {
+  var scene = theScene;
+  var rpcController = null;
+  var sourceTargetName;
+  var appNameTargetRegistered = false;
+  var functions = {};
+  var onRPCCall;
+
+  this.registerApp = function(targetName) {
```

Comment:
I wonder if something like appNamespace or appId might be a better name for this var than "targetName".